packages/strapi-connector-bookshelf/lib/mount-models.js

```diff
@@ -413,12 +428,29 @@ module.exports = async ({ models, target }, ctx, { selfFinalize = false } = {})
 
           // Hack Bookshelf to create a many-to-many polymorphic association.
           // Upload has many Upload_morph that morph to different model.
+          const populateFn = qb => {
+            qb.where(qb => {
```

Comment:
shouldn't it be a orWhere ?

Reply:
good catch, it's a typo !

Reply:
No, it's more than that, I messed when moving the code in the function